src/drivers/clock/Mybuild

```diff
@@ -69,6 +69,18 @@ module mips_clk extends embox.arch.clock {
 	@NoRuntime depends embox.kernel.time.timer_handler
 }
 
+module riscv_clk extends embox.arch.clock {
+	source "riscv_clk.c"
+
+	depends embox.kernel.irq
+	depends embox.kernel.time.clock_source
+	depends embox.arch.interrupt
+	@NoRuntime depends embox.kernel.time.timer_handler
+
+	option number base_mtime=0x200bff8
+	option number base_mtimecmp=0x2004000
```

Comment:
Can you please provide document for this?

Reply:
https://sifive.cdn.prismic.io/sifive%2Fc89f6e5a-cf9e-44c3-a3db-04420702dcc1_sifive+e31+manual+v19.08.pdf
page 24
6.1 CLINT Memory Map

Reply:
Thank you.